### PR TITLE
CQT 39: check if ANTLR4 dependency is still required (otherwise, remove)

### PR DESCRIPTION
**Pull request by rturrado**
Remove ANTLR4 dependency.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -80,7 +79,7 @@ asyncio_mode = "auto"
 
 [tool.coverage.run]
 branch = true
-omit = ["tests/*", "**/.tox/**", "opensquirrel/parsing/antlr/generated/*"]
```
Comment (elenbaasc) [inline pyproject.toml:83]: Good catch 😄

**Review by elenbaasc — COMMENTED** [review]
Do you know if we need to update the `poetry.lock` file and commit the changes, before merging?

**Comment by rturrado** [comment]
> Do you know if we need to update the `poetry.lock` file and commit the changes, before merging?

I don't think `poetry.lock` should be part of the codebase, should it?

**Comment by rturrado** [comment]
> > Do you know if we need to update the `poetry.lock` file and commit the changes, before merging?
> 
> I don't think `poetry.lock` should be part of the codebase, should it?

It seems it should: https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control

I've removed the ANTLR4 package from `poetry.lock` as well.

**Review by elenbaasc — APPROVED** [review]
Cool! Can be merged :)

**Pull request merged**